README.md

```diff
@@ -7,14 +7,11 @@ For a high level summary of MIE and its use cases, please read, [How to Rapidly
 # Installation
 
 
-You can deploy MIE in your AWS account with the following Cloud Formation templates:
+You can deploy MIE in your AWS account with the following Cloud Formation template:
 
-Region| Launch
-------|-----
-US East (N. Virginia) | [![Launch in us-east-1](doc/images/launch-stack.png)](https://console.aws.amazon.com/cloudformation/home?region=us-east-1#/stacks/new?stackName=mie&templateURL=https://rodeolabz-us-east-1.s3.amazonaws.com/media_insights_engine/v2.0.1/cf/media-insights-stack.template)
-US West (Oregon) | [![Launch in us-west-2](doc/images/launch-stack.png)](https://console.aws.amazon.com/cloudformation/home?region=us-west-2#/stacks/new?stackName=mie&templateURL=https://rodeolabz-us-west-2.s3.us-west-2.amazonaws.com/media_insights_engine/v2.0.1/cf/media-insights-stack.template)
```

Comment:
Are we removing the us-west-2 one click deploy link?

Reply:
Yes. The us-east-1 deploy link will work for any region. So the UX goes like this:

1. User clicks on the one and only deploy link.
2. This opens the cloud formation template in the us-east-1 region.
3. User clicks region drop-down in the upper-right-corner of the AWS console if they prefer a different region. The same us-east-1 template will work for that region.

click on the one and only deploy button

Reply:
This is the same way solution builder works.  I will, however, remove "us-east-1" text from the deploy HTML button.